apps/recnet/src/app/not-found.tsx

```diff
@@ -1,7 +1,8 @@
 import { cn } from "@/utils/cn";
```

Comment:
[nit] change the filename to camel case

Reply:
Ahh, that makes sense but actually this is kind of a frontend convention to name this file like this.

Ref:
- https://akhilaariyachandra.com/blog/using-clsx-or-classnames-with-tailwind-merge
- https://www.youtube.com/watch?v=re2JFITR7TI
- https://korayguler.medium.com/how-to-merge-react-and-tailwind-css-class-names-f5faeb10ed24